### ISSUE TITLE
Printing issues

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord-dev/ts-android#1640]
There are several problems with printing to pdf. (This could be made an Epic and individual issues created for each problem described herein)
1. Page numbering should not begin until the beginning of the actual document text.
2. The \v n verse markers should be replaced with just the verse number with no space between it and the text for that verse.
3. The book title should appear only on the very first page, and not on the page directly after the TOC.
4. The chapter heading should appear at the top of the page on which that chapter's text begins, and not on its own page.
5. After doing a merge and resolving a conflict in the first chunk of a Jude project, the pdf verses do not appear in the correct order as shown here:
![image](https://cloud.githubusercontent.com/assets/8171759/19315829/df7fb630-9064-11e6-9438-54fc762d4399.png)

**Comment by PhotoNomad0** [comment]
@cckozie - looking at #5 - did you save that to the cloud so I can look at the contents after the merge?

**Comment by neutrinog** [comment]
related to #1659

**Comment by cckozie** [comment]
The #5 problem above has been fixed as of build 144.

**Comment by PhotoNomad0** [comment]
@cckozie - are the other issues fixed (#1-#4)?

**Comment by cckozie** [comment]
See issue 1805 - failed to print project imported from usfm.

**Comment by cckozie** [comment]
These issues are now all fixed as of travis build 1144. There are now some other issues with printing that I will write up in a new issue.
